src/gsHSplines/gsTHBSplineBasis.h

```diff
@@ -127,6 +127,9 @@ class gsTHBSplineBasis : public gsHTensorBasis<d,T>
     /// @brief Gives back the basis at a slice in \a dir_fixed at \a par
     BoundaryBasisType * basisSlice(index_t dir_fixed,T par ) const;
 
+    // Look at gsBasis class for documentation
+    void active_into(const gsMatrix<T>& u, gsMatrix<index_t>& result) const;
```

Comment:
To have a correct over-ride of the function, in the gsHTensorBasis.h file the gsHTensorBasis::active_into function should be declared as virtual.